conformance/null.ion

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2023-2024 Amazon.com, Inc.  All rights reserved.
```

Comment:
Can we leave out the dates from the copyright header? We've been using this header for `ion-java` and `ion-rust`, which both also use the Apache-2.0 license.
```suggestion
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
```

Reply:
Thanks, I didn't realize the recommendations had changed.